### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by FL303**
Added more context and linked to learning path.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,65 @@
 # Humanitec Google Cloud Reference Architecture Implementation
+> TL;DR
+>
+> [Follow this tutorial to spin up your Humanitec Google Cloud Reference Architecture Implementation](README.md#how-to-spin-up-your-humanitec-google-cloud-reference-architecture-implementation)
```
Comment (TobiasBabin) [inline README.md:4]: Make it more clear that we are staying in the _same_ tutorial here, like "Skip the theory and go straight to spinning up your Humanitec Google Cloud Reference Architecture Implementation".

File: README.md [hunk 2]
```diff
@@ -1,13 +1,65 @@
...
+>
+> [Follow this learning path to master you Internal Developer Platform](https://developer.humanitec.com/training/master-your-internal-developer-platform/introduction/)
 
+Building an Internal Developer Platform (IDP) can come with many challenges. To give you a head start, we’ve created a set of [reference architectures](https://humanitec.com/reference-architectures) based on hundreds of real-world setups. These architectures described in code provide a starting point to build your own IDP within minutes, along with customization capabilities to ensure your platform meets the unique needs of your users (developers).
+
+The initial version of this reference architecture has been presented by Marco Marulli, Principal Delivery Lead II, McKinsey & Company at [PlartformCon 2023](https://www.youtube.com/watch?v=TswbmEXrvJg).
```
Comment (TobiasBabin) [inline README.md:10]: Typo in "PlatformCon"

**Review by TobiasBabin — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (FL303) [reply README.md]: Fixed

**Review by TobiasBabin — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -60,7 +112,7 @@ This reference architecture implementation uses Terraform. You will need to do t
    terraform apply
    ```
```
Comment (TobiasBabin) [inline README.md:114]: Add a heading  `## Terraform docs` like in the [AWS README](https://github.com/humanitec-architecture/reference-architecture-aws/blob/38d045a397b39672fd3e6ef2e6a9b965ec201d62/README.md?plain=1#L132)

**Review by TobiasBabin — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (FL303) [reply README.md:114]: Added, please check

**Review by TobiasBabin — APPROVED** [review]

**Pull request merged**